cmd/builder/internal/builder/main_test.go

```diff
@@ -432,7 +433,7 @@ func verifyGoMod(t *testing.T, dir string, replaceMods map[string]bool) {
 		}
 
 		_, ok := replaceMods[req.Mod.Path]
-		assert.Truef(t, ok, "Module missing from replace statements list: %s", req.Mod.Path)
+		assert.Truef(t, ok, "Module missing from replace statements list: %s. ", req.Mod.Path)
```

Comment:
Is this change necessary?
```suggestion
		assert.Truef(t, ok, "Module missing from replace statements list: %s", req.Mod.Path)
```